app/controllers/applies_controller.rb

```diff
@@ -49,13 +22,12 @@ def deny
     @mission.save
 
     authorize @mission
-
     redirect_to mentor_profil_path
   end
 
   private
 
-  def mission_params
-    params.require(:mission).permit(:start_date, :end_date, :mission_id)
+  def apply_params
+    params.require(:apply).permit(:status, :motivation, :mission_id, :padawan_id)
```

Comment:
I don't think we need the "status" and "padawan_id" in the params since status is generated automatically and padawan_id will be the current user.